workspace_hack/Cargo.toml

```diff
@@ -16,26 +16,23 @@ publish = false
 [dependencies]
 ahash = { version = "0.7", features = ["std"] }
 anyhow = { version = "1", features = ["backtrace", "std"] }
-bstr = { version = "0.2", features = ["lazy_static", "regex-automata", "serde", "serde1", "serde1-nostd", "std", "unicode"] }
 bytes = { version = "1", features = ["serde", "std"] }
-chrono = { version = "0.4", features = ["clock", "libc", "oldtime", "serde", "std", "time", "winapi"] }
+chrono = { version = "0.4", features = ["clock", "iana-time-zone", "js-sys", "oldtime", "serde", "std", "time", "wasm-bindgen", "wasmbind", "winapi"] }
```

Comment:
I am not entirely up to speed what do workspace hacks do but having `wasm-bindgen`, `wasmbind` and `winapi` show up as features does seem weird. Was this expected?

Reply:
https://crates.io/crates/cargo-hakari guess this would be the best explanation

As far as I understand it, it tries to unit all crates across all project under minimal common feature subset, so that the same crate does not get recompiled multiple times due to different features used.

Reply:
Resolved this already but seems that they are just the defaults:

https://github.com/chronotope/chrono/blob/f60893a0dce544ac6013038b6def0bddd562b1ca/Cargo.toml#L19 

which means it comes from pageserver and compute_tools.